.github/workflows/build-ironic-images.yaml

```diff
@@ -86,3 +137,13 @@ jobs:
           fail_on_unmatched_files: true
           files: ironic-images/upload/*
         if: ${{ github.ref == 'refs/heads/main' }}
+
+  cleanup:
+    runs-on: ubuntu-latest
+    needs: build-tenant-images
+
+    steps:
+      - name: Clean up artifacts
+        uses: geekyeggo/delete-artifact@v5
```

Comment:
Is this just so we don't have a ton of artifacts?

Reply:
Yep exactly. I think the default artifact lifetime is 90 days.

Reply:
We can set the retention time to 1 day but I won't block this going in.